lib/src/boringssl/bindings/ffigen.yaml

```diff
@@ -0,0 +1,53 @@
+name: WebCryptoDartDL
+description: 'Bindings to src/webcrypto_dart_dl.h.'
+output: 'lib/src/boringssl/bindings/generated_bindings.dart'
+headers:
+  entry-points:
+    - src/webcrypto_dart_dl.h
+    - src/webcrypto.c
+compiler-opts: '-Ithird_party/dart-sdk/src/runtime/ -Ithird_party/boringssl/src/include -I/Library/Developer/CommandLineTools/usr/lib/clang/12.0.0/include'
```

Comment:
Can `-I/Library/Developer/CommandLineTools/usr/lib/clang/12.0.0/include` be put in a platform specific option somewhere?

Reply:
https://pub.dev/packages/ffigen/changelog 2.3.0 makes this obsolete.

Reply:
Awesome! :D